### PR TITLE
Finish guides, add images

### PR DESCRIPTION
**Pull request by wrightmartin**
This breaks the guide prototype into three and adds links between them

### CODE REVIEW

File: reduce-tta.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <li>Central Team don’t have a good enough understanding of information held by each Service Area. <a href="#">Build an information asset register</a> and provide it to Central Team. Keep it updated. Ideally <a href="#">build it in to the software</a>.</li>
+        <li>Champions aren’t trained enough to know that sitting on a request causes overall problems. No consequences for doing this. <a href="#">Improve training rollout</a> and <a href="#">consider introducing rewards and incentives for good performance</a>.
+          </li>
+      </ul>
+      <h2 class="heading-medium">Champion to Information Holder Delays</h2>
+      <ul class="list list-bullet">
+        <li>Not enough Champions – they’re responsible for too broad an area and don’t have enough knowledge of all the information they may have access to. Read ‘<a href="#">how many champions do we need?</a>’
+          </li>
+        <li>Information Holders aren’t trained enough to know that sitting on a request causes overall problems. No consequences for doing this. a href="#">Improve training rollout</a> and <a href="#">consider introducing rewards and incentives for good performance</a>.</li>
```
Comment (garethrees) [inline reduce-tta.html]: First url is broken

File: reduce-tta.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  <div class="column-two-thirds">
+    <h1 class="heading-xlarge">
+      <span class="heading-secondary">Guide</span>
+      How to reduce time to allocate
+    </h1>
+  </div>
+</div>
+<div class="grid-row">
+  <div class="column-two-thirds">
+      <h2 class="heading-medium">How to Improve</h2>
```
Comment (garethrees) [inline reduce-tta.html]: Not sure we need this heading now its on a separate page?

File: reduce-tta.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: "Time to allocate"
```
Comment (garethrees) [inline reduce-tta.html]: "How to reduce time to allocate"?

File: measure-cycle-time.html [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+      <img src="{{site.baseurl}}/assets/img/lead-time-1.png" class="limit-image" />
+      <p>Lead time is comprised of Time to Allocate and Cycle Time. There’s more flexibility to target a specific area if we break up Lead Time in to its components.</p>
+      <img src="{{site.baseurl}}/assets/img/lead-time-2.png" class="limit-image" />
+      <p>Each process is roughly independent so a delay in either area can contribute to a longer Lead Time.</p>
+      <img src="{{site.baseurl}}/assets/img/lead-time-3.png" class="limit-image" />
+      <p>Cycle Time consists of many discrete tasks with predictable timescales, and a few unpredictable but critical tasks such as information retrieval.</p>
+      <p>Time to Allocate is “wasted” time as it's not directly contributing to the response. Reducing it provides more time for the “Cycle Time”, and could even build in some buffer time.</p>
+      <img src="{{site.baseurl}}/assets/img/lead-time-4.png" class="limit-image" />
```
Comment (garethrees) [inline measure-cycle-time.html]: Worth using the updated images where the 20-day timeline is the same proportion https://docs.google.com/drawings/d/1DY0hzvCYM2GKLuhiw25efFMXI7mUPJ59rxKCzd91_Bc/edit?

File: measure-cycle-time.html [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+layout: default
+title: "Time to allocate"
+bodyclass: ""
+---
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <h1 class="heading-xlarge">
+      <span class="heading-secondary">Guide</span>
+      How to measure cycle time
```
Comment (garethrees) [inline measure-cycle-time.html]: How to measure _time to allocate_?

File: measure-cycle-time.html [hunk 6]
```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: "Time to allocate"
```
Comment (garethrees) [inline measure-cycle-time.html]: Should reflect main header?

**Review by garethrees — CHANGES_REQUESTED** [review]
Looks good! Couple of minor tweaks.

Would also be worth adding the `not-implemented` class to any links that don't go anywhere so that its clear that they're examples rather than broken.

File: reduce-tta.html (hunk 1 above)
Comment (wrightmartin) [reply reduce-tta.html]: I've added `class="not-implemented"` to the fake links now

**Pull request merged**